nservicebus/scheduling/index.md

```diff
@@ -13,9 +13,11 @@ The Scheduler is a lightweight/non-durable API that helps schedule a task that n
 
 {{WARNING:
 
-Scheduling a task depends on [delayed delivery](/nservicebus/messaging/delayed-delivery.md). If the delayed delivery operation fails, the Scheduler will be interrupted and a `ScheduledTask` message will be forwarded to the error queue. When this happens the scheduled task will stop executing unless the `ScheduledTask` message is retried.
+Under certain failure scenarios, the scheduled task can stop firing until the endpoint is restarted. 
```

Comment:
Also, why remove the information that it relies on delayed delivery?

Reply:
I think it makes the message hidden in the details. The messages is:
> Scheduler is not fully-reliable.